### PR TITLE
[Terraform] updated gcp terraform script

### PR DESCRIPTION
**Pull request by wulfthimm**
#### What it does
This improves the installation of Gitpod on GCP. The installtion of GCP full and the installer are using the same Terraform modules. Adding a module creates the cloud and kubernetes resources exporting the according values for the Gitpod installation. They are passed to the Gitpod installation.

### CODE REVIEW

**Comment by csweichel** [comment]
before this is merged, please squash into one commit!

File: chart/values.yaml [hunk 1]
```diff
@@ -328,11 +328,11 @@ components:
       http:
         expose: true
         containerPort: 80
-        nodePort: 31080
+        # nodePort: 31080
       https:
         expose: true
         containerPort: 443
-        nodePort: 31443
+        # nodePort: 31443
```
Comment (meysholdt) [inline chart/values.yaml:335]: why remove?

**Review by meysholdt — APPROVED** [review]
LGTM. please rebase before merging.

File: chart/values.yaml (hunk 1 above)
Comment (ArthurSens) [reply chart/values.yaml:335]: If needs to be removed, can't we just erase the line? :grimacing:

**Comment by wulfthimm** [comment]
> LGTM. please rebase before merging.
Thanks, has been rebased.

**Pull request merged**